senders/pushover/pushover.go

```diff
@@ -91,7 +95,17 @@ func (sender *Sender) buildMessage(events moira.NotificationEvents, throttled bo
 }
 
 func (sender *Sender) buildTitle(events moira.NotificationEvents, trigger moira.TriggerData) string {
-	return fmt.Sprintf("%s %s %s (%d)", events.GetSubjectState(), trigger.Name, trigger.GetTags(), len(events))
+	title := fmt.Sprintf("%s %s %s (%d)", events.GetSubjectState(), trigger.Name, trigger.GetTags(), len(events))
+	tags := 1
+	for len(title) > titleLimit-30 {
+		var tagBuffer bytes.Buffer
+		for i := 0; i < len(trigger.Tags)-tags; i++ {
+			tagBuffer.WriteString(fmt.Sprintf("[%s]", trigger.Tags[i]))
+		}
+		title = fmt.Sprintf("%s %s %s and %d more tags (%d)", events.GetSubjectState(), trigger.Name, tagBuffer.String(), tags, len(events))
```

Comment:
Let's change this logic:
In your solution you reserve 30 symbols for ```and %d more tags``` notation but in typical cases tag len is short (examples: [FreeSpace] or [Datacenter]). Lets cut off tags silently

Reply:
Oh, let's use ```...``` instead of ```and %d more tags```